pkg/ride/constraints.go

```diff
@@ -1,26 +1,46 @@
 package ride
 
 import (
+	"math"
+
 	"github.com/pkg/errors"
 	"github.com/wavesplatform/gowaves/pkg/ride/ast"
 )
 
 const (
-	MaxChainInvokeComplexityV3V4 = 4000
-	MaxChainInvokeComplexityV5   = 26000
-	MaxChainInvokeComplexityV6   = 52000
+	maxChainInvokeComplexityV3V4   = 4000
+	maxChainInvokeComplexityV5     = 26000
+	maxChainInvokeComplexityV6     = 52000
+	maxAssetVerifierComplexityV1V2 = 2000
+	maxAssetVerifierComplexityV3V6 = 4000
+	maxVerifierComplexity          = 2000
+	unlimitedVerifierComplexity    = math.MaxInt16
 )
 
-func maxChainInvokeComplexityByVersion(version ast.LibraryVersion) (int, error) {
+func MaxChainInvokeComplexityByVersion(version ast.LibraryVersion) (int, error) {
 	// libV1 and libV2 don't have callables
 	switch version {
 	case ast.LibV3, ast.LibV4:
-		return MaxChainInvokeComplexityV3V4, nil
+		return maxChainInvokeComplexityV3V4, nil
 	case ast.LibV5:
-		return MaxChainInvokeComplexityV5, nil
+		return maxChainInvokeComplexityV5, nil
 	case ast.LibV6:
-		return MaxChainInvokeComplexityV6, nil
+		return maxChainInvokeComplexityV6, nil
 	default:
 		return 0, errors.Errorf("unsupported library version %d", version)
 	}
 }
+
+func MaxAssetVerifierComplexity(v ast.LibraryVersion) int {
+	if v > ast.LibV2 {
+		return maxAssetVerifierComplexityV3V6
+	}
+	return maxAssetVerifierComplexityV1V2
+}
+
+func MaxVerifierComplexity(rideV5Activated bool) int {
+	if rideV5Activated {
+		return maxVerifierComplexity
+	}
+	return unlimitedVerifierComplexity
+}
```

Comment:
Yep, but we don't have those heights in our blockchain settings files. And on Stagenet this height is set after the activation of RideV6, I think, accidentally because no such transactions were found on Stagenet and Testnet. 
If I'll found such transaction on Mainnet I'll add the setting and set them.

Reply:
Won't change this because MainNet was imported correctly.